docs/GetStarted/createProfileACM.md

```diff
@@ -77,6 +77,9 @@ A Profile provides configuration information to the AMT Firmware during the acti
    
 6. Provide or generate a strong **MEBX Password**. This password can be used to access Intel® Manageability Engine BIOS Extensions (Intel® MEBX) on the AMT device.
 
+    !!! note
+        By default for both **AMT Password** and **MEBX Password** auto generation checkboxes are tikced. To enter static passwords please un-tick the respective auto generation checkboxes.
```

Comment:
By default both **AMT Password** and **MEBX Password** auto generation checkboxes are selected (checked). To enter static passwords, unselect the respective auto generation checkboxes.

Reply:
Updated